tests/integration_tests/test_applications.py

```diff
@@ -189,6 +189,15 @@
         ]
     ],
     # Database
+    "get_file_from_db": [
+        ["--file_name", "mirror_CTA-S-LST_v2020-04-07.dat"],
+        ["--file_name", "PSFcurve_data_v2.txt"],
+        [
+            "--file_name",
+            "ray-tracing-North-LST-1-d10.0-za20.0_validate_optics\
```

Comment:
You should break this string differently to be consistent with how we do it everywhere else. It should be

```
(
"ray-tracing-North-LST-1-d10.0-za20.0_validate_optics"
".ecsv"
)
```

(ignore the indentation I wrote above, just the fact that it's closed within parenthesis and without the `\`

Reply:
I break now for different lists of arguments, it should be more readable. Please have a look